### ISSUE TITLE
Add example projects

### ISSUE THREAD

**Issue by thornbill** [jellyfin/jellyfin-sdk-typescript#186]
Provide usage examples for the following project types:

- [ ] basic web project
- [ ] web project with bundler
- [ ] react native project
- [ ] node.js project?

**Comment by anultravioletaurora** [comment]
I imagine you'd prefer smaller project examples as opposed to larger ones?

**Comment by thornbill** [comment]
Yeah the idea was to provide just the bare minimum to get people started. Like [this](https://github.com/thornbill/jf-sdk-ts-tracker/blob/master/apps/listing-unstable/listing.mjs).
